docs/package.json

```diff
@@ -33,11 +33,11 @@
     "@glimmer/tracking": "^1.0.4",
     "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^3.0.0",
-    "ember-animated": "0.12.0",
-    "ember-animated-tools": "github:ember-animation/ember-animated-tools#1edcfd7f0be0a9b4b69450f0fc4d945f31277215",
+    "ember-animated": "workspace:*",
+    "ember-animated-tools": "file:../ember-animated-tools-0.5.0.tgz",
     "ember-auto-import": "^2.4.0",
     "ember-cli": "~4.1.1",
-    "ember-cli-addon-docs": "^4.2.1",
+    "ember-cli-addon-docs": "file:./ember-cli-addon-docs-4.2.2.tgz",
```

Comment:
Couldn't just use branch from https://github.com/ember-learn/ember-cli-addon-docs/pull/1164 as had to remove `prepare` script otherwise it was trying to link some packages which was failing under `ember-animated`.

We actually may need to restructure `ember-cli-addon-docs` into monorepo to get rid from that `scripts/link-them.sh` but it's another story.